target/linux/ramips/mt7621/config-5.15

```diff
@@ -12,9 +12,6 @@ CONFIG_CLKSRC_MIPS_GIC=y
 CONFIG_CLK_MT7621=y
 CONFIG_CLOCKSOURCE_WATCHDOG=y
 CONFIG_CLONE_BACKWARDS=y
-CONFIG_CMDLINE="rootfstype=squashfs,jffs2"
```

Comment:
Deleting already enabled options/features from the common mt7621 kernel config sounds dangerous at the first glance. Are you sure that the removed options are not needed by any of the mt7621 devices?

Reply:
> Deleting already enabled options/features from the common mt7621 kernel config sounds dangerous at the first glance. Are you sure that the removed options are not needed by any of the mt7621 devices?

Yes, this would affect them all and it's not what I wanted to do at the end of the day. This is left here to illustrate what I would like to achieve but I cannot come up with a solution where you have kernel flags set differently for one device only.  The vendor u-boot passes kernel params "root=ubi0:rootfs_0 ubi.mtd=ubi rootfstype=ubifs mtdparts=MT7621-NAND:1M(uboot),-(ubi)". I tried creating a subtarget under mt7621, but that did not work out.

That said, do you have suggestions on how I should approach this?